src/core/constants.js

```diff
@@ -195,8 +196,17 @@ export const INSTALL_ERROR = 'INSTALL_ERROR';
 
 // Tracking categories.
 export const INSTALL_CATEGORY = 'AMO Addon / Theme Installs';
```

Comment:
Do we still need this? Won't it always be either `INSTALL_THEME_CATEGORY` or `INSTALL_EXTENSION_CATEGORY`?

Reply:
this was set up this way so the "general" category could be passed in and then depending on the type, the more descriptive category was used. I'll take another look tho to see about cleaning this up.